clubs.html

```diff
@@ -164,21 +199,20 @@ <h3 class="sub-club">Completely Kids</h3>
           </div>
           <div class="col-xs-12 col-sm-4">
 
-            <img class="club-logo" src="img/clubs/AIM-Logo.png">
+            <img class="club-logo" src="img/clubs/completely-kids-logo.jpg">
             <address>
               1905 Harney St<br>
               Omaha, NE 68102
             </address>
           </div>
           <div class="col-xs-12 col-sm-4">
-            <h4>Tuesdays, Wednesdays 4-6pm</h4>
-            <p>6th-8th grade</p>
-            <p><span>Instructors:</span> Not Announced</p>
+            <h4>Tuesdays, Thursdays 4:30-5:30pm</h4>
+            <p>6th grade</p>
+            <p><span>Instructors:</span> Erin Lasiter</p>
```

Comment:
Didn't know Erin was going to do a club too! Awesome. Let's be sure to get pics and bios for the AIM and Do Space people for the volunteers page too. I don't know if we need these by tomorrow, but we can do it soonish. We could put the current instructors in the top rows in that page as well.